### ISSUE TITLE
Clearer log message for invalid ip for cluster manager when Riak starts

### ISSUE THREAD

**Issue by mk1s** [basho/riak_repl#394]
If the cluster_mgr ip is invalid currently the log message is:

```
013-08-31 15:33:33.188 [error] <0.11584.0> Supervisor riak_core_cluster_mgr_sup had child riak_core_service_mgr started with riak_core_service_mgr:start_link() at undefined exit with reason {'EXIT',{{badarg,invalid_ip},[{riak_core_service_mgr,start_link,1,[{file,"src/riak_core_service_mgr.erl"},{line,105}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,303}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,287}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,253}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,227}]}]}} in context start_error
2013-08-31 15:33:33.188 [error] <0.11572.0> Supervisor riak_repl_sup had child riak_core_cluster_mgr_sup started with riak_core_cluster_mgr_sup:start_link() at undefined exit with reason shutdown in context start_error
```

It would nice to have a clear distinct message for troubleshooting misconfigurations.

**Comment by cmeiklejohn** [comment]
Addressed by #470.

**Comment by cmeiklejohn** [comment]
Closing; resolved.
